examples/nlp/language_modeling/bert_pretraining.py

```diff
@@ -404,9 +404,11 @@ def create_pipeline(data_file, batch_size, preprocessed_data=False, batches_per_
 # callback which prints training loss and perplexity once in a while
 if not args.only_mlm_loss:
     log_tensors = [train_loss, mlm_loss, nsp_loss]
```

Comment:
How about changing its name to train_log_tensors to be consistent?
They are named train_tensors and eval_tensors in the scripts I worked on.